clipper_admin/clipper_manager.py

```diff
@@ -586,16 +596,18 @@ def deploy_predict_function(self,
         predict_function : function
             The prediction function. Any state associated with the function should be
             captured via closure capture.
-        labels : list of str
-            A set of strings annotating the model
         input_type : str
             One of "integers", "floats", "doubles", "bytes", or "strings".
+        labels : list of str, optional
+            A list of strings annotating the model
         num_containers : int, optional
             The number of replicas of the model to create. More replicas can be
             created later as well. Defaults to 1.
 
         Example
         -------
+        Define a feature function ``center()`` and train a model on the featurized input::
```

Comment:
That's an intentional pydoc markup symbol (https://docs.python.org/devguide/documenting.html#source-code)